Assets/Scenes/MainEventScene.unity

```diff
@@ -9250,11 +9459,11 @@ RectTransform:
   m_Father: {fileID: 1888291109}
   m_RootOrder: 9
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0.44, y: 0.175}
-  m_AnchorMax: {x: 0.56, y: 0.22}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 0, y: 0}
-  m_Pivot: {x: 0.5, y: 0.5}
+  m_AnchorMin: {x: 0.5, y: 0.5}
```

Comment:
This is good, but would using anchors here be helpful?

Reply:
I double checked to make sure that removing the anchors for the error text and button don't affect the different aspect ratios. I feel like we'll probably be rearranging/updating the panels when we work on improving aesthetics, so I can always update all of the canvas objects to use anchors :).